apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +137,71 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor name of the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "The days of the week for when the course meets",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description:
+        "The type of course level of the course (e.g. Lower Division, Upper Division, or Graduate)",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "The type of section of the course (e.g. Lecture, Discussion, Lab)",
+      example: "Lec",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   fullCourses: z
     .enum(fullCoursesOptions)
     .optional()
+    .openapi({
+      description: "The enrollment status of the course",
+      example: "SkipFull",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
-  cancelledCourses: z.enum(cancelledCoursesOptions).optional(),
-  units: z.optional(z.literal("VAR").or(z.string())),
-  startTime: timeSchema.optional(),
-  endTime: timeSchema.optional(),
+  cancelledCourses: z.enum(cancelledCoursesOptions).optional().openapi({
+    description: "Include or exclude cancelled courses",
+    example: "Exclude",
+  }),
+  units: z.optional(z.literal("VAR").or(z.string())).openapi({
+    description: "The number of units for the courses",
```

Comment:
Same comment as above; you might want to say something like "Filter courses by unit count". You should also explain how "VAR" is handled to fully explain this parameter. Take a look at the [source code](https://github.com/icssc/anteater-api/blob/2250280db9d667be91299d74d67fbdc0ad589aec/apps/api/src/services/websoc.ts#L201) to learn what it does.